CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - Add the new `go.opentelemetry.io/contrib/instrgen` package to provide auto-generated source code instrumentation. (#3068, #3108)
+- Add support additional semantic conventions `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp`. (#????)
```

Comment:
Is a changelog entry needed? It seems like this is just moving code around. From the user perspective nothing changed, right?

Reply:
Removed.